app/scaffold-controllers/navigationController.js

```diff
@@ -11,17 +13,26 @@ function(
     Application,
     AnchoredLayoutPlugin,
     NavigationPlugin,
-    NavigationHeaderController
+    NavigationHeaderController,
+    SearchBarController,
+    SearchConfig
```

Comment:
Would like other's opinions on just incorporating `SearchConfig` into the `SearchBarController` itself. Then all users of the `SearchBarController` would have the same `uiSource` and `queryUrl`? I think this would make the usage of the `SearchBarController` cleaner but then wouldn't be able to support configuring different uses of the `SearchBarController` per app. What do the reviewers think?